src/h3/api/_api_template.py

```diff
@@ -17,22 +17,42 @@ def _api_functions(
         _globals,
 ):
     def versions():
-        return c.versions()
+        """ Return version numbers for both the Python library
+        and the underlying H3 C library.
+
+        Returns
+        -------
+        dict
+        """
+        return _a.versions()
 
     def string_to_h3(h):
-        return c.hex2int(h)
+        """ Converts a hexadecimal string to an H3 64-bit integer index
+        """
+        return _a.hex2int(h)
 
     def h3_to_string(x):
-        return c.int2hex(x)
+        """ Converts an H3 64-bit integer index to a hexadecimal string
+        """
+        return _a.int2hex(x)
 
     def num_hexagons(resolution):
-        return c.num_hexagons(resolution)
+        """ Return the total number of *cells* (hexagons and pentagons)
+        for the given resolution.
+
+        todo: is that right?
```

Comment:
Yes, you can confirm this by checking that the number at res 0 is 122.

Reply:
cool!